arrayOperationTask/script.js

```diff
@@ -0,0 +1,33 @@
+// reverse
+const reverseFun = (data)=>{
+   let reverserArr = []
+   let datalength= data.length
+   for(let i=0;i<datalength;i++){
+      let poppedItem = data.pop()
+      reverserArr.push(poppedItem)
+   }
+   return reverserArr
+}
+// multiply by 5
+const multiplyFun = (data)=>{
+   data = data.map((item)=>{

```

Comment:
Good stuff with your use of the `map` function here @eligifty-coder.
An even shorter way to write this would be:
```const multiplyFun = (data) => data.map((item) => item * 5)```

Reply:
Also you always want your functions to have good and descriptive names.
What do you think of a name like `multiplyByFiveFunc` vs `multiplyFun`?

Reply:
The changes specified have been effected.